### ISSUE TITLE
Adding a song

### ISSUE THREAD

**Issue by blake00** [AurousGroup/Bug-Tracker#13]
I can't add a song to my collection the plus buttom doesn't work

Windows 10 pro 86x

**Comment by haydster7** [comment]
I am operating on Windows 8.1 64-bit and when I also click the "+" icon next to the play icon nothing happens. RIght clicking on the song also yields no results.

**Comment by gregor160300** [comment]
You first need to set at least one folder in settings where your music is/will be stored
Then when you click the + the song is saved to your music folder or whichever folder you set in settings

**Comment by Thelodger** [comment]
Indeed, the + button does not add the song to the collection I set in the settings.

**Comment by gregor160300** [comment]
Maybe it randomly works then xD

**Comment by navjotjsingh** [comment]
Can confirm. It does not work.

**Comment by benoobie** [comment]
Try add it to a playlist, i have had it work that way when it fails using the + button.

**Comment by NoShotz** [comment]
The plus button adds it to your library/downloads it, you need to click the
••• button

**Comment by blake00** [comment]
It simply does not work properly, please fix it.
